openssl.conf

```diff
@@ -47,3 +48,4 @@ subjectKeyIdentifier = hash
 authorityKeyIdentifier = keyid,issuer:always
 keyUsage = critical, digitalSignature, keyEncipherment
 extendedKeyUsage = serverAuth
+subjectAltName=DNS:localhost
```

Comment:
Multiple subjectAltName values are possible, either by
```
subjectAltName = DNS:localhost, DNS:lvh.me, IP:127.0.0.1, IP:127.0.0.2
```
or 
```
subjectAltName = @alt_names

[alt_names]
DNS.1 = localhost
DNS.2 = lvh.me
IP.1 = 127.0.0.1
IP.2 = 127.0.0.2
```

My thinking is that this list of domains and IP addresses should be

#### 1) The following, by default
```
DNS.1 = localhost
DNS.2 = lvh.me
IP.1 = 127.0.0.1
```
Use case: I'm working on an app that can be hosted on a single domain, and wish to develop with HTTPS on localhost.

#### 2) Added to, machine-wide at the developer's discretion

Use case: I have a static IP address or host name on my corporate network, and want ANY devcert certificate to facilitate TLS connections for that IP/hostname.

#### 3) Added to, by request of the requestor of a certificate (and ideally at the explicit consent of the machine owner), on a per-certificate basis

Use case: I work on an app that involves multiple subdomains, even in development (i.e. we are building something like *.github.io, where users create their own subdomains)